app/src/main/java/com/bogdan/codeforceswatcher/model/Contest.kt

```diff
@@ -0,0 +1,19 @@
+package com.bogdan.codeforceswatcher.model
+
+import android.arch.persistence.room.Entity
+import android.arch.persistence.room.PrimaryKey
+import com.google.gson.annotations.SerializedName
+import retrofit2.Call
+import retrofit2.http.GET
+
+@Entity
+data class Contest(@PrimaryKey val id: Long, val name: String, @SerializedName("startTimeSeconds") val time: Long, val phase: String)
+
+data class ContestResponse(val status: String, val result: List<Contest>)
+
+interface ContestApi {
```

Comment:
You don't need (but you can, of course) an API class for every network request. IMHO it's better to have smth like `CodeforcesApi` and include all requests to Codeforces there.

Then you need to extract entities (`Contest`, `ContestResponse`) to separate classes.